### PR TITLE
base_country_state_translatable Module

### PR DESCRIPTION
**Pull request by rami-wafaie**

### CODE REVIEW

File: base_country_state_translatable/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# © initOS GmbH 2017
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Translate Country States",
+    "version": "10.0.1.0.0",
+    "depends": [],
```
Comment (pedrobaeza) [inline base_country_state_translatable/__manifest__.py]: Should depend on base

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by rami-wafaie** [comment]
Thanks for the remark, Dependency is corrected

**Review by chienandalu — APPROVED** [review]
Looks good :+1:

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@cubells please review

**Review by cubells — APPROVED** [review]
:+1: 

LGTM

**Comment by mart-e** [comment]
Hello,

FYI, we have added the flag `translate=True` in `l10n_multilang` in 11.0 at odoo/odoo@bc84eb20

**Pull request merged**